controllers/articles.controllers.js

```diff
@@ -1,4 +1,13 @@
-const { fetchArticleById } = require('../models/articles.models')
+const { fetchArticles, fetchArticleById } = require('../models/articles.models')
+
+exports.getArticles = async (req, res, next) => {
+    try {
+        const fetchedArticles = await fetchArticles()
+        res.status(200).send(fetchedArticles)
```

Comment:
Send back an object with the articles inside it, when you change this you will have to alter your test. It won't be on body anymore but body.fetchedArticles if that's the key you send it on!